core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletMetadata.java

```diff
@@ -597,6 +602,18 @@ private void setLocationOnce(String val, String qual, LocationType lt, boolean s
     location = new Location(val, qual, lt);
   }
 
+  private void setOperationIdOnce(String val, boolean suppressError) {
+    if (location != null) {
```

Comment:
Could add check that operation Id is null also.

```suggestion
    if (location != null || operationId != null) {
```

Could add a similar check of the operationId in the earlier method `setLocationOnce()`.  May need to adjust the erros messages in each method.

Reply:
`operationId` is not used in `setLocationOnce()`. Should we still check that it is not null there? Also, are you suggesting that the existing error messages be changed if the `operationId != null` is added? If so how might they be changed?

Reply:
The operation id and location are mutually exclusive, we should never see a tablet with both set.  That is why I was thinking of adding something to the set location method.  When we set a location, its expected that the operation id is null.  When we set an operation id, its expected that the location is nuill.

Was thinknig th error message could include the location and operationid if both are being checked.

Reply:
I tried to fix this in 84e15bd